pages/DetailsTrip/[city]/index.js

```diff
@@ -96,14 +96,17 @@ export default function DetailsOfTrip() {
         <StyledH4>Hotels:</StyledH4>
         <StyledBlox>
           <StyledUl>
-            {city.hotels.map((hotel) => (
-              <StyledList key={uuidv4()}>
-                <StyledSpan>{hotel.hotel}</StyledSpan>
-                {hotel.hotelPrice ? (
-                  <StyledSpan>{hotel.hotelPrice} Euro</StyledSpan>
-                ) : null}
-              </StyledList>
-            ))}
+            {city.hotels.map(
+              (hotel) =>
+                hotel.hotel !== "" && (
```

Comment:
I don't know why you could get hotels with the empty `hotel` property, in this case it would be better to filter them out before mapping:

```js
{city.hotels.filter(hotel => !!hotel.hotel).map(({ id, hotel, hotelPrice }) => (
  <StyledList key={id}>
    <StyledSpan>{hotel}</StyledSpan>
    {hotelPrice ? (
      <StyledSpan>{hotelPrice} Euro</StyledSpan>
    ) : null}
  </StyledList>
))}
```